example.json

```diff
@@ -13,7 +13,7 @@
 "USE_IPXE":false,
 "USE_HTTP":false,
 "USE_TFTP":true,
-"MODE_DEBUG":false,
+"MODE_DEBUG":"",
```

Comment:
`"dhcp" in args.MODE_DEBUG.lower()` for example won't work before as it's a bool. This value should be a string now.